### PR TITLE
set ASPSMS as SMS provider for 2fa

### PR DESCRIPTION
**Pull request by Swiss-Devs**
Dear Rick,

we finally have completed the new tutorial for 2fa with ASPSMS.

Best regards from Switzerland, Till

### CODE REVIEW

**Comment by GuardRex** [comment]
Purely as an curious bystander, I was just wondering why Twilio is being dropped here? If that's a NosyRex:tm: question, just ignore me ... I can take a hint. :wink:

**Comment by Rick-Anderson** [comment]
@Swiss-Devs  Thanks, I'll take a look at this tomorrow.

File: aspnetcore/security/authentication/2fa.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ uid: security/authentication/2fa
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/2fa.md]: You can add yourself as co-author

File: aspnetcore/security/authentication/2fa.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ uid: security/authentication/2fa
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/2fa.md]: Can you remove the warning? You did test this with RTM bits - right?

File: aspnetcore/security/authentication/2fa.md (hunk 1 above)
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/2fa.md]: @Swiss-Devs 
I think you're better off creating a new project using Visual Studio 2017 RC4  and replace the old project.

File: aspnetcore/security/authentication/2fa.md (hunk 2 above)
Comment (Swiss-Devs) [inline aspnetcore/security/authentication/2fa.md]: We have successfully tested it with VS 2017 RC4 / Core 1.0/1.1 and converted the Warning to a Note.

File: aspnetcore/security/authentication/2fa.md (hunk 1 above)
Comment (Swiss-Devs) [inline aspnetcore/security/authentication/2fa.md]: We have added ourselves as co-author.

Comment (Swiss-Devs) [inline aspnetcore/security/authentication/2fa.md]: Can you please be more specific regarding your statement "creating a new project"? We don't know exactly what to do.

File: aspnetcore/security/authentication/2fa.md [hunk 3]
```diff
@@ -14,14 +14,14 @@ uid: security/authentication/2fa
 ---
 # Two-factor authentication with SMS
 
->[!WARNING]
-> This page documents version 1.0.0-beta8 and has not yet been updated for version 1.0.0
+> [!NOTE]
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/2fa.md]: Remove the note. We only add warnings when content is out of date. The assumption is that the document works with the current version.

**Comment by Rick-Anderson** [comment]
@Swiss-Devs  never mind, I removed the note for you. Thanks for your help.

**Comment by Rick-Anderson** [comment]
Fixes #2744 
Fixes #2293

**Pull request merged**